training-infra/templates/buckets-creation.yaml

```diff
@@ -28,7 +28,7 @@ spec:
           value: {{ .Values.environment }}
       restartPolicy: Never
   backoffLimit: 4
-  activeDeadlineSeconds: 120  # Job will fail after 120 seconds if not completed
+  activeDeadlineSeconds: 360  # Job will fail after 120 seconds if not completed
```

Comment:
you probalby want to update these comments. Though a better question is whether these defaults can be kept?